pages/sponsors.tsx

```diff
@@ -33,19 +33,19 @@ export default function SponsorsPage() {
         <div className="l-container md:mt-10 md:py-12">
           <div className="flex flex-wrap">
             <div className="w-full text-center mb-2 md:mb-0 mt-4 md:mt-0 items-center justify-center">
-              <h3 className="title-w black pt-10 md:pt-0">
+              <h3 className="title black pt-10 md:pt-0">
                 Why Sponsor droidconke
               </h3>
             </div>
           </div>
           <div className="mx-auto px-4 md:py-16">
             <div className="items-center flex flex-wrap mb-4 md:mb-10">
               <div className="w-full md:w-4/12 ml-auto px-2 md:px-8 mt-4 md:mt-0">
-                <div className="text-center md:text-left">
+                <div className="text-center md:text-left text-xl md:text-2xl">
```

Comment:
`text-xl md:text-2xl` should be in h3 not here, juu ina increase hadi ,P> which it should not